iterators.tex

```diff
@@ -348,7 +348,7 @@
   concept bool MoveWritable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<Out>@\newtxt{()}@ &&
       requires (Out o, T v) {
-        *o = move(v);
+        *o = move(v); @\newtxt{// not equality preserving}@
```

Comment:
As the most conservative position since we don't constrain the value or type of that expression to be in any way useful. This could just as well be `void(*o = move(v))` (or `void(*o = v)` for Writable below).